app/configs/fatalitiesListViewColumns.tsx

```diff
@@ -25,34 +25,54 @@ export const fatalitiesListViewColumns: ColDataCardDef<fatalitiesListRow>[] = [
     sortable: true,
   },
   {
-    path: "law_enforcement_ytd_fatality_num",
-    label: "Law Enforcement Number",
+    path: "ytd_fatal_crash",
+    label: "YTD Fatal Crash",
     sortable: true,
   },
   {
-    path: "ytd_fatal_crash",
-    label: "YTD Fatal Crashes",
+    path: "ytd_fatality",
+    label: "YTD Fatality",
     sortable: true,
   },
   {
-    path: "ytd_fatality",
-    label: "YTD Fatalities",
+    path: "law_enforcement_ytd_fatality_num",
+    label: "LE #",
     sortable: true,
   },
   {
     path: "crash_date_ct",
     label: "Crash Date",
     sortable: true,
     valueFormatter: formatDate,
+    style: { whiteSpace: "nowrap" },
   },
   {
     path: "location",
     label: "Location",
     sortable: true,
+    style: { whiteSpace: "nowrap" },
   },
   {
     path: "victim_name",
     label: "Victim Name",
     sortable: true,
+    style: { whiteSpace: "nowrap" },
+  },
+  {
+    path: "recommendation.atd__recommendation_status_lkp.rec_status_desc",
+    label: "FRB Status",
+    style: { whiteSpace: "nowrap" },
+    sortable: true,
+  },
+  {
+    path: "recommendation.rec_text",
+    label: "FRB Recommendation",
+    style: { minWidth: "400px" },
```

Comment:
it's a weird quirk of the `table-responsive` class that our only option here is to set `minWidth` rather than `width` or `maxWidth`. maybe someone else can figure it out but i gave up 😞